README.rst

```diff
@@ -81,6 +82,9 @@ Features
   live grepping.
   See the wiki for `a sample configuration <https://github.com/Julian/lean.nvim/wiki/Configuring-&-Extending#live-grep>`_.
 
+* If `telescope.nvim <https://github.comnvim-telescope/telescope.nvim>`_ is present a `:Telescope loogle` command
```

Comment:
Minor typos, and an extra `_` will make the nonsense cryptic reST error go away:

```suggestion
* If `telescope.nvim <https://github.com/nvim-telescope/telescope.nvim>`__ is present a `:Telescope loogle` command is available as a frontend for the `Loogle <https://loogle.lean-lang.org>`_ JSON API.
```